source/features/closing-remarks.tsx

```diff
@@ -61,14 +73,14 @@ function addExistingTagLink(tagName: string): void {
 		);
 	}
 
+	const linkedTag = <a href={tagUrl} className="text-mono text-small">{tagName}</a>;
```

Comment:
Not enough attention?

```suggestion
	const linkedTag = <a href={tagUrl} className="Link--primary text-bold">{tagName}</a>;
```

<img width="863" alt="image" src="https://user-images.githubusercontent.com/44045911/231423077-2be78a9a-7003-43fb-a0a5-61cb15beb2f2.png">